### PR TITLE
Another feature

### PR DESCRIPTION
**Pull request by FilipePizzoloTR**
Include new files

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1 @@
+var texto = 'abc';
```
Comment (FelipeHSilvaTR) [inline script.js:1]: Error 01

File: poc.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Why do we use it?
+It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
```
Comment (FelipeHSilvaTR) [inline poc.txt]: Error 02

**Review by FelipeHSilvaTR — CHANGES_REQUESTED** [review]
Fix errors

File: poc.txt (hunk 2 above)
Comment (FilipePizzoloTR) [reply poc.txt]: Fixed script.

File: script.js (hunk 1 above)
Comment (FilipePizzoloTR) [reply script.js:1]: Fixed txt.

**Review by FelipeHSilvaTR — APPROVED** [review]
Fixed errors. Congratulations.

**Pull request merged**